src/pages/SKills/Skills.jsx

```diff
@@ -14,16 +14,8 @@ function Skills() {
         >
           <h2>Skills</h2>
         </motion.div>
-        <motion.div
-          animate={{ x: "2%" }}
-          initial={{ x: "-10%" }}
-          transition={{ ease: "easeOut", duration: 0.9 }}
-          whileHover={{ scale: 1.2 }}
-        >
-          <SkillsDescription />
-        </motion.div>
-      </div>
-      <div className="margin">
+        <SkillsDescription />
+        <br />
```

Comment:
<br /> are used for line breaks, not to add space between components: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br